docs/index.md

```diff
@@ -43,7 +43,7 @@ Install Codacy on an existing cluster using our Helm chart:
         !!! important
             **If you are using MicroK8s** you don't need to install kubectl because you will execute all `kubectl` commands as `microk8s.kubectl` commands instead. To simplify this, [check how to create an alias](infrastructure/microk8s-quickstart.md#notes-on-installing-codacy) for `kubectl`.
 
-    -   [Helm client](https://v2.helm.sh/docs/using_helm/#installing-helm) version 2.16.3
+    -   [Helm](https://helm.sh/docs/intro/install/) version 3.2.0
```

Comment:
Could we omit the patch number? I'm asking this because the `snap intall` does not allow specifying the patch number, so at some point customers will have versions greater than 3.2.0.

```suggestion
    -   [Helm](https://helm.sh/docs/intro/install/) version 3.2
```